packages/core/src/components/highlight-box/highlight-box.scss

```diff
@@ -2,7 +2,11 @@
 @use "~dso-toolkit/src/variables/units";
 @use "~dso-toolkit/src/components/highlight-box";
 
+@include utilities.box-sizing();
```

Comment:
@tfrijsewijk ff uitzoeken

Reply:
dit moet nog volgens mij ;)

Reply:
Ja, ga ik doen! 🙏

Reply:
Hier zie ik nog steeds spacing issues in Percy

Reply:
Uitgezocht en waar mogelijk opgelost. Een paar situaties heb ik als verbetering geaccepteerd, voornamelijk waren dat situaties waar margins niet meer collapsen